sources/settings-extensions/ntp/src/lib.rs

```diff
@@ -85,4 +87,22 @@ mod test {
         let results = serde_json::to_string(&ntp).unwrap();
         assert_eq!(results, test_json);
     }
+
+    fn test_options_ntp() {
```

Comment:
In order for Rust to recognize this as a unit test, you need to provide the `#[test]` macro here:

```suggestion
    #[test]
    fn test_options_ntp() {
```